components/automate-chef-io/content/docs/data-lifecycle.md

```diff
@@ -30,16 +30,16 @@ Navigate to _Settings_ > _Data Lifecycle_ and adjust any settings you would like
 
 Users with `dataLifecycle:*` IAM access are able to see the data lifecycle job statuses, configure jobs, or run jobs.
 
-![](/images/docs/data-lifecycle.png)
+![Data Lifecycle](/images/docs/data-lifecycle.png)
 
 ### Event Feed
 
-The event feed data lifecycle settings allow you to remove all event feed data and Chef Infra Server actions after a set amount of days.
+The Event Feed Data Lifecycle settings allow you to remove all event feed data and Chef Infra Server actions after a set amount of days.
 The default is to remove event feed data after 7 days, and Chef Infra Server actions after 30 days.
 
 ### Service Groups
 
-The service group data lifecycle settings allow you to label health check reports as disconnected and automatically remove them after a set amount of time.
+The Service Group Data Lifecycle settings allow you to label health check reports as disconnected and automatically remove them after a set amount of time.
 The default is to label health check reports as disconnected after 5 minutes, and remove disconnected services after 5 days.
```

Comment:
ALSO! I noticed that the screenshot in this page has label health check reports as disconnected after 5 minutes, and remove disconnected services after **7** days. Should we amend this line to say 7 days? The Periodic Removal section in applications-dashboard.md also references a 7 day default.

Reply:
Good catch! It is 7 days: https://github.com/chef/automate/blob/7da2e48913e6616e815278b1392f7df916e788e5/components/applications-service/pkg/server/periodic.go#L474

Reply:
Added!